dcos-log/router/middleware.go

```diff
@@ -1,10 +1,64 @@
 package router
 
-import "net/http"
+import (
+	"fmt"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/gorilla/mux"
+)
 
 func authMiddleware(next http.Handler) http.Handler {
+	client := &http.Client{
+		Timeout: time.Second * 3,
+		Transport: &http.Transport{
+			DisableKeepAlives: true,
+		},
+	}
+
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		// ... auth code here
+		// get request variables
+		authorization := r.Header.Get("Authorization")
+		frameworkID := mux.Vars(r)["framework_id"]
+		executorID := mux.Vars(r)["executor_id"]
+		containerID := mux.Vars(r)["container_id"]
+
+		// in order to validate a request we need authorization header and mux variables:
+		// framework_id, executor_id, container_id. If we miss any of it we just serve the request.
```

Comment:
this has to work in open and EE DC/OS. In open we dont have Authorization header. But we still have to serve it